File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        set<int> n(nums.begin(), nums.end());
        int a=nums.size();
        int b=n.size();
        if(a==n){
            return false;
        }else{
            return true;
        }
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
# class Solution(object):
#     def containsDuplicate(self, nums):
#         for x in nums:
#             if nums.count(x)>1:
#                 return True
#             # else:
#             #     return False
#         return False


class Solution(object):
    def containsDuplicate(self, nums):
        nums.sort()
        for i in range(1,len(nums)):
            if nums[i]==nums[i-1]:
                return True
            # else:
            #     return False
        return False
